### DISCUSSION TITLE
Document ID in input component

### DISCUSSION THREAD

**Discussion by matt416** [sanity-io/sanity#3914] [Sanity Studio Support]
How would I get the document ID within an input component? I was expecting a hook like useClient, but I can't find anything that has the ID

**Comment by andrilla-francis** [comment]
I would love an answer to this as well. I feel like this should be simple, but I can't find any documentation or solutions for this.

**Comment by andrewmumblebee** [comment]
https://www.sanity.io/docs/studio-react-hooks#5cb6622b6381

```
import {useFormValue} from 'sanity'

// inside input component
const id = useFormValue(['_id'])
```
